src/condition.ts

```diff
@@ -10,7 +10,7 @@ import { Format } from "./format";
 
 export class Condition {
 	type: ConditionType;
-	message: string;
+	_message: string | Function;
```

Comment:
The function type can be more specific: `_message: string | () => string`

Reply:
Change made, the function had to be in parenthesis when used in a union type.